api/src/main/resources/descriptions/io.strimzi.api.kafka.model.authentication.KafkaClientAuthenticationOAuth.adoc

```diff
@@ -0,0 +1,93 @@
+To use OAuth client authentication, set the `type` property to the value `oauth`.
+OAuth authentication can be configured in several different ways.
+
+.Client ID and secret
+
+You can configure the address of your OAuth server in the `tokenEndpointUri` property together with the OAuth client ID and client secret.
```

Comment:
```suggestion
You can configure the address of your authorization server in the `tokenEndpointUri` property together with the client ID and client secret.
```

Reply:
We use 'authorization server' in the main doc. And where we say OAuth, we say OAuth 2.0. I took out the OAuth from before the client ID as we only refer to 'Kafka client' in the main doc.